src/replay_stage.rs

```diff
@@ -449,6 +433,40 @@ impl ReplayStage {
         let next_slots = blocktree.get_slots_since(&[slot_index]).expect("Db error");
         next_slots.first().cloned()
     }
+
+    fn forward_entries(
+        entries: &[Entry],
+        slot: u64,
+        slot_leader: Pubkey,
+        bank: &Arc<Bank>,
```

Comment:
Is it possible to implement this without a dependency on the Bank? Perhaps making use of `SlotMeta::last_index`?